pallets/schemas/src/lib.rs

```diff
@@ -381,11 +445,229 @@ pub mod pallet {
 		) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
 
-			let schema_id = Self::create_schema_for(model, model_type, payload_location, settings)?;
+			let (schema_id, _) =
+				Self::create_schema_for(model, model_type, payload_location, settings, None)?;
 
 			Self::deposit_event(Event::SchemaCreated { key: sender, schema_id });
 			Ok(())
 		}
+
+		/// Propose to create a schema.  Creates a proposal for council approval to create a schema
+		///
+		#[pallet::call_index(5)]
+		#[pallet::weight(
+			match schema_name {
+				Some(_) => T::WeightInfo::propose_to_create_schema_v2(model.len() as u32),
+				None => T::WeightInfo::propose_to_create_schema(model.len() as u32)
+			}
+		)]
+		pub fn propose_to_create_schema_v2(
```

Comment:
re: comment above, I can seem people reading this extrinsic as "this is to create a v2 schema, where is the one to propose creating a v3 schema?"

Reply:
so it's important to note these versions are the versions for extrinsics not the underlying structure. We do not have a schema v2 or schema v3 structure. I think that would make sense if we were doing lazy migrations only.